File: TemperatureConverter.py
# Temperature Converter
# - Convert temperature between Celsius and Fahrenheit.
# - Formulas:
#   - C → F : (C * 9/5) + 32
#   - F → C : (F - 32) * 5/9
# - Constraints:
#   - Temperature values can be any real number.

class Solution:
    def celsiusToFahrenheit(self, celsius: float) -> float:
        return (celsius * 9/5) + 32

    def fahrenheitToCelsius(self, fahrenheit: float) -> float:
        return (fahrenheit - 32) * 5/9

File: TemperatureConverter.cs
// Temperature Converter
// - Convert temperature between Celsius and Fahrenheit.

using System;

class Solution
{
    // Celsius to Fahrenheit
    public double CelsiusToFahrenheit(double celsius)
    {
        return (celsius * 9.0 / 5.0) + 32.0;
    }

    // Fahrenheit to Celsius
    public double FahrenheitToCelsius(double fahrenheit)
    {
        return (fahrenheit - 32.0) * 5.0 / 9.0;
    }
}
